aurora-mysql-plugin/pom.xml

```diff
@@ -99,8 +99,8 @@
           <instructions>
             <_exportcontents>
               io.cdap.plugin.auroradb.mysql.*;
-              io.cdap.plugin.db.batch.source.*;
-              io.cdap.plugin.db.batch.sink.*;
+              io.cdap.plugin.db.source.*;
+              io.cdap.plugin.db.sink.*;
               org.apache.commons.lang;
```

Comment:
Not related to this PR, but these are some interesting exports. I don't see why these should be needed, but they can definitely cause issues with other plugins. Normally we should only need to export packages from the project itself.

Can you open a jira to investigate whether we can remove these exports?

Reply:
Created JIRA : https://cdap.atlassian.net/browse/PLUGIN-1521 to track the said issue.